src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -98,7 +105,27 @@ public int getHttpStatus() {
   }
 
   public boolean isSuccess() {
-    return successCodes.contains(getHttpStatus());
+    boolean isSuccess = false;
+    // ServiceNow Rest API may return 200 OK response with an error message in the response body.
+    // Normally we would expect non-200 response code if there's an error.
+    if (getRetryable()) {
+      throw new RetryableException();
+    } else if (successCodes.contains(getHttpStatus())) {
+      isSuccess = true;
+    }
+    return isSuccess;
+  }
+
+  private boolean checkRetryable() {
```

Comment:
Return value is not used, can return void

Reply:
Changed the return type to void. 2c4538f24a45de8576bb7a51b2422ad070a696dd